lib/client-assets.php

```diff
@@ -133,6 +133,28 @@ function gutenberg_override_style( $handle, $src, $deps = array(), $ver = false,
 	wp_register_style( $handle, $src, $deps, $ver, $media );
 }
 
+/**
+ * Registers all the WordPress packages scripts that are in the standardized
+ * `build/` location.
+ *
+ * @since 4.5.0
+ */
+function gutenberg_register_packages_scripts() {
+	$packages_dependencies = include dirname( __FILE__ ) . '/packages-dependencies.php';
+
+	foreach ( $packages_dependencies as $handle => $dependencies ) {
+		$package_name = substr( $handle, 3 );
```

Comment:
This is a bit different than what's in core? Core defines the dependencies handles omitting the `wp-` and adds it back later. Is this intended to be an improvement, and if so is it planned to be brought back to core?

At the very least, it's very unclear what we're doing with the `substr` by just looking at this code, and it could do for a comment.

Reply:
Will add a comment. I will open a ticket in Trac to change that to align with the plugin so we could copy and paste whenever something gets updated. I wanted to have both keys and dependencies use the same format for consistency.

Reply:
I refactored to this:

```php
// Remove `wp-` prefix from the handle to get the package's name.
$package_name = strpos( $handle, 'wp-' ) === 0 ? substr( $handle, 3 ) : $handle;
```

There is also a check which should make it easier to reason when we need to remove the prefix.